node/src/BaseClient.ts

```diff
@@ -523,6 +523,7 @@ export class BaseClient {
     /** Retrieve the values of multiple keys.
      * See https://redis.io/commands/mget/ for details.
      *
+     * @remarks When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slot.
```

Comment:
Now it's hash slot without the graves. Whichever style is correct, we should be consistent throughout.